.travis.yml

```diff
@@ -1,16 +1,14 @@
 language: generic
-sudo: false
+# sudo shouldn't be required, but currently tests fail when run in a container
+# on travis instead of a vm. See https://github.com/dask/distributed/pull/1563.
```

Comment:
The tornado PR linked from this one was closed as it appeared that Travis was fixed; should this workaround be reverted?

Reply:
No, the workaround is still needed. That PR was closed as we were done debugging.

Reply:
Not the distributed PR, the [tornado one](https://github.com/tornadoweb/tornado/pull/2199).

Reply:
Looks like travis no longer has containerized environments, so I suspect that this could be removed. However, since the `sudo` setting no longer changes the build time it's not really a high priority to change it. Feel free to submit a PR if you want to.